### ISSUE TITLE
`GetRootMediaFolder` returns null if the current user has the `ManageMediaContent` permission.

### ISSUE THREAD

**Issue by csymons** [OrchardCMS/Orchard#6074]
```
System.NullReferenceException was unhandled by user code
  HResult=-2147467261
  Message=Object reference not set to an instance of an object.
  Source=Orchard.MediaLibrary
  StackTrace:
       at Orchard.MediaLibrary.Controllers.AdminController.RecentMediaItems(Int32 skip, Int32 count, String order, String mediaType)
       at lambda_method(Closure , ControllerBase , Object[] )
       at System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
       at System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
       at System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.<BeginInvokeSynchronousActionMethod>b__39(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
       at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
       at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3d()
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass46.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3f()
  InnerException: 

```

**Comment by jean** [comment]
This happens when adding an `Image` part to a new page layout, select Ordered by "recently created" in the Media Library, and select "Recent".

**Comment by sfmskywalker** [comment]
The `Image` part is supposed to be attached to just the **Image** part, and not to be added as an element to a page layout. Did you manually turn the Image part into a placeable one?

**Comment by csymons** [comment]
Ah actually it was fixed in #5938
